modules/otel-config-instrumentation.adoc

```diff
@@ -98,6 +98,105 @@ spec:
 
 |===
 
+[id="otel-instrumentation-options_exporter_{context}"]
+== Exporter configuration
+
+Although the `Instrumentation` custom resource supports setting up one or more exporters per signal, auto-instrumentation configures only the OTLP Exporter. So you must configure the endpoint to point to the OTLP Receiver on the Collector.
+
+.Sample exporter TLS CA configuration using a config map
```

Comment:
```suggestion
.Sample exporter TLS CA configuration using a config map
```
Wondering if config map is an object, a la `config map`? Or is it just a "configuration YAML file"?